cmd/saturn-l2/main.go

```diff
@@ -103,6 +110,8 @@ var (
 
 	// DNS Hostname of Saturn L1 Nodes
 	saturn_l1_hostName = "strn.pl"
+
+	defaultOrchestratorURL = "https://orchestrator.strn.pl/nodes/nearby"
```

Comment:
Nit: The orchestrator url is `https://orchestrator.strn.pl`, the discovery url is `https://orchestrator.strn.pl/nodes/nearby`. Therefore I think either we rename this variables to `defaultL1DiscoveryUrl` or change the env var to not need the path suffix. What do you think?

Reply:
What is the discovery URL for the test net? Is it a different hostname with the same path, e.g. something like `https://orchestrator.test.strn.pl/nodes/nearby`? In that case providing a host name only may work.

In general, I feel it's better to avoid magic in configuration. If we need to configure discovery URL, then the config option should specify the full URL. 

This way it's much easier to mock the discovery service for testing, as the mock does not need to replicate the path structure of the real one.

Reply:
The orchestrator test net url is https://orchestrator.saturn-test.network/nodes/nearby (from https://github.com/filecoin-saturn/orchestrator#readme)